bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormConstants.java

```diff
@@ -34,6 +34,9 @@ private FormConstants() {
     /** The resource type for text input v1 */
     public static final String RT_FD_FORM_TEXT_V1 = RT_FD_FORM_PREFIX + "textinput/v1/textinput";
 
+    /** The resource type for date picker v1 */
+    public static final String RT_FD_FORM_DATE_PICKER_V1 = RT_FD_FORM_PREFIX + "datepicker/v1/datepicker";
```

Comment:
Can we remove this constant and use it directly where it is required ? With constants for resource type, this file would grow exponentially

Reply:
not exponentially. just 20 lines. I'd suggest to keep it as it is.